kubernetes/samples/scripts/domain-lifecycle/README.md

```diff
@@ -0,0 +1,65 @@
+### Domain lifecycle sample scripts
+The WebLogic Server Kubernetes Operator project provides a set of sample scripts to shut down or start a specific managed-server, cluster or the entire domain. These can be helpful when scripting the lifecycle of a WebLogic Server Domain.
```

Comment:
I have a general concern that readers that reach this point in the documentation might be very new to WKO and so may not realize that they need to have deployed a domain first and that the domain's introspector must have run successfully first.  Please call out these assumptions here.

Reply:
LGTM